content/docs/architecture/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Architecture"
+---
+The Orchestrator architecture comprises several integral components, each contributing to the seamless execution and management of workflows. Illustrated below is a breakdown of these components:
+
+- Backstage: Serving as the primary interface, Backstage fulfills multiple roles:
+  - Orchestrator Plugins: Both frontend and backend plugins are instrumental in presenting deployed workflows for execution and monitoring.
+  - Notifications Plugin: Employs notifications to inform users or groups about workflow events.
+- Sonataflow Operator: This controller manages the Sonataflow custom resource (CR), where each CR denotes a deployed workflow.
+- Sonataflow Runtime: As a deployed workflow, Sonataflow Runtime is currently managed as a Kubernetes (K8s) deployment by the operator. It operates as an HTTP server, catering to requests for executing workflow instances. Within the Orchestrator deployment, each Sonataflow CR corresponds to a singular workflow. However, outside this scope, Sonataflow Runtime can handle multiple workflows. Interaction with Sonataflow Runtime for workflow execution is facilitated by the Orchestrator backend plugin.
+- Data Index Service: This serves as a repository for workflow definitions, instances, and their associated jobs. It exposes a GraphQL API, utilized by the Orchestrator backend plugin to retrieve workflow definitions and instances.
+- Job Service: Dedicated to orchestrating scheduled tasks for workflows.
+- OpenShift Serverless: This operator furnishes serverless capabilities essential for workflow communication. It employs Knative eventing to interface with the Data Index service and leverages Knative functions to introduce more intricate logic to workflows.
+- AMQ Streams (Strimzi/Kafka): While not presently integrated into the deployment's current iteration, this operator is crucial for ensuring the reliability of the eventing system.
+- KeyCloak: Responsible for authentication and security services within applications. While not part of the current deployment version, it is essential for enhancing security measures.
+
+![Architecture Diagram](./architecture-diagram.png)
```

Comment:
can you increase the size a bit?

Reply:
couldn't find a direct way of doing so, need to research a bit and maybe enable some hook to support it:
https://gohugo.io/getting-started/configuration-markup/#image-markdown-example

Reply:
Ok, we can address this later :)